fenix/app/src/androidTest/java/org/mozilla/fenix/experimentintegration/conftest.py

```diff
@@ -67,12 +67,13 @@ def gradlewbuild(gradlewbuild_log):
 @pytest.fixture(name="experiment_data")
 def fixture_experiment_data(experiment_url):
     data = requests.get(experiment_url).json()
-    for item in data["branches"][0]["features"][0]["value"]["messages"].values():
-        item["surface"] = "homescreen"
-        item["style"] = "URGENT"
-        for count, trigger in enumerate(item["trigger"]):
-            if "USER_EN_SPEAKER" not in trigger:
-                del item["trigger"][count]
+    if data["branches"][0]["features"][0]["value"].get("messages"):
+        for item in data["branches"][0]["features"][0]["value"]["messages"].values():
+            item["surface"] = "homescreen"
+            item["style"] = "URGENT"
+            for count, trigger in enumerate(item["trigger"]):
+                if "USER_EN_SPEAKER" not in trigger:
+                    del item["trigger"][count]
```

Comment:
LGTM. 

Nit: If you'd like, I'd recommend instead of index based element access `0` here, pretty sure you can just `iter()` and `next()` 

branch_iter = iter(data["branches"])
first_branch = next(branch_iter, None)

In this example, first_branch will be the first element of data["branches"], or None if the list is empty